alembic/versions/c3da223e60d8_add_has_backup_field.py

```diff
@@ -0,0 +1,47 @@
+"""add has_backup field
+
+Revision ID: c3da223e60d8
+Revises: d0aa961845b9
+Create Date: 2023-09-22 11:10:14.616339
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from alembic import op
+from cg.constants.sequencing import Sequencers
+from cg.store.models import Flowcell
+
+# revision identifiers, used by Alembic.
+revision = "c3da223e60d8"
+down_revision = "d0aa961845b9"
+branch_labels = None
+depends_on = None
+
+Session = sessionmaker()
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    bind = op.get_bind()
+    session = Session(bind=bind)
+
+    op.add_column(
+        table_name="flowcell", column=sa.Column("has_backup", sa.Boolean(), nullable=False)
```

Comment:
Does this work? If we add a new column all the current entries will have value null right, which is in violation with `nullable=False`. If this part is tested please ignore the comment

Reply:
It was tested on stage and went well 🙂 The field has a default of `False` and the code just below updates the values.